src/parsers/dict_to_struct.jl

```diff
@@ -115,8 +115,8 @@ Takes a string or symbol "name" and returns a list of devices within a collectio
 function _get_device(name::Union{String,Symbol}, collection, devices = [])
     if isa(collection,Array) && !isempty(collection) && isassigned(collection)
             fn = fieldnames(typeof(collection[1]))
-            if :name in fn
-                [push!(devices,d) for d in collection if d.name == name]
+            if :name in fn || :base in fn
+                [push!(devices,d) for d in collection if PowerSystems.name(d) == name]
```

Comment:
Note that this approach means that we will likely see tons of name collisions, and so will have to prefix function calls with "PowerSystems.".

Reply:
I must be missing something, why do we have to prefix `PowerSystems.` to the `name` function?

Reply:
We only have to do it when name collides with a local variable, as it does here.  Other possibilities are to change the local variable to something like "name_" or call the function "get_name".

Reply:
Of course, my bad. I'm fine with renaming the local variable here.